custom_components/tuya_local/devices/earu_3phase_breaker.yaml

```diff
@@ -0,0 +1,199 @@
+name: EARU 3-Phase Breaker Meter
```

Comment:
This should be generic, all branding belongs under products.

Example "Breaker meter"